src/MatBlazor/Components/MatDrawer/BaseMatDrawer.cs

```diff
@@ -47,17 +47,23 @@ public bool Opened
         [Parameter]
         public EventCallback<bool> OpenedChanged { get; set; }
 
+        [Parameter]
+        public string? Width { get; set; } = null;
 
         private DotNetObjectReference<BaseMatDrawer> dotNetObjectRef;
         public BaseMatDrawer()
         {
-            
             ClassMapper
                 .Add("mdc-drawer")
                 .Add("mat-drawer")
                 .If("mdc-drawer--dismissible", () => Mode == MatDrawerMode.Dismissible)
                 .If("mdc-drawer--modal", () => Mode == MatDrawerMode.Modal);
 
+            if (Width != null)
```

Comment:
Use this instead of check with `if` 

    StyleMapper
        .GetIf(() => $"--mat-drawer-custom-width: {Width}", () => !string.IsNullOrWhiteSpace(Width));

Reply:
Done, but in `BaseMatDrawerContainer.cs`